liboptv/src/parameters.c

```diff
@@ -147,6 +150,13 @@ track_par* read_track_par(char *filename) {
     FILE* fpp;
     track_par *ret = (track_par *) malloc(sizeof(track_par));
     
+/* @WARNING: This is really important, some libraries (e.g. ROS, Qt4) seems to set the 
+system locale which takes decimal commata instead of points which causes the file input 
+parsing to fail */    
+    setlocale(LC_ALL, "C"); 
+    setlocale(LC_NUMERIC,"C");
+    setlocale(LC_ALL, "POSIX");
```

Comment:
I don't know why LC_ALL is needed, but if that's what works...
In any case, you set it here twice. Only the most recent one applies. And IIRC, setting LC_ALL also means that you don't need to set LC_NUMERIC, because ALL is all...
If LC_NUMERIC works, I suggest to restrict to changing only that.

Reply:
got it. changed to `setlocale(LC_NUMERIC,"C");` only